rector.out

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
What is this file?

Reply:
This is the output of running `vendor/bin/rector process --dry-run`.  It shows what Rector would change if we removed `--dry-run`.  In this particular case, it is used as an indicator that nothing needs to change (afaik) to use this codebase in PHP 8.  It's also nice to have around for a diff of a future run of Rector.